### ISSUE TITLE
sfdx force:source:deploy does not deploy multiple packages in sequence mentioned in sfdx-project.json

### ISSUE THREAD

**Issue by Gurudata** [forcedotcom/cli#565]
### Steps To Reproduce:
I have a below folder structure  where Package2 comes before Package1
PackageDirectory
   ---Package2
  ---Package1

My sfdx-project.json has the following sequence where Package1 comes before Package2
     Package1
    Package2

When I run the below command to deploy multiple packages, Package2 is deployed before Package1. The sequence in sfdx-project.json is ignored.
sfdx force:source:deploy -p PackageDirectory -u devOrg

### Expected result
sfdx force:source:deploy should respect package sequence in sfdx-project.json, Package1 should be deployed before Package2

### Actual result
sfdx force:source:deploy follows the sequence of packages in the root package directory and not the sequence in sfdx-project.json

**Comment by amphro** [comment]
@Gurudata is this using salesforce-alm@mpd version?

**Comment by Gurudata** [comment]
> @Gurudata is this using salesforce-alm@mpd version?

Yes @amphro  Please see my version details below.
 ```
$sfdx plugins --core
@oclif/plugin-autocomplete 0.1.5 (core)
@oclif/plugin-commands 1.3.0 (core)
@oclif/plugin-help 3.1.0 (core)
@oclif/plugin-not-found 1.2.4 (core)
@oclif/plugin-plugins 1.9.0 (core)
@oclif/plugin-update 1.3.10 (core)
@oclif/plugin-warn-if-update-available 1.7.0 (core)
@oclif/plugin-which 1.0.3 (core)
@salesforce/sfdx-trust 3.2.1 (core)
analytics 1.12.0 (core)
generator 1.1.3 (core)
salesforce-alm 49.3.1-4 (mpd)
salesforcedx 49.0.7 (pre-release)
├─ salesforcedx-templates 48.10.0
└─ @salesforce/sfdx-plugin-lwc-test 0.1.5
sfdx-cli 7.70.1 (core)
```

**Comment by amphro** [comment]
This has been fixed and MPD is now released in `49.10.0`
